contrib/PanopticSeg/tools/train.py

```diff
@@ -104,7 +104,25 @@ def parse_train_args(*args, **kwargs):
         default='gpu',
         choices=['cpu', 'gpu', 'xpu', 'npu', 'mlu'],
         type=str)
+    parser.add_argument(
+        '--profiler_options',
+        type=str,
+        help='The option of train profiler. If profiler_options is not None, the train ' \
```

Comment:
`'The option of train profiler. If profiler_options is not None, the train profiler is enabled. Refer to the paddleseg/utils/train_profiler.py for details.'`
->
```
"Options of the training profiler. If `profiler_options` is not None, the training profiler will be enabled. Refer to `paddleseg/utils/train_profiler.py` for details."
```